src/audio/tables/samplebank_table.c

```diff
@@ -0,0 +1,50 @@
+#include "attributes.h"
+#include "z64audio.h"
+
+// Symbol definition
+
+extern AudioTable gSampleBankTable;
+#pragma weak gSampleBankTable = sSampleBankTableHeader
+
+// Externs for table
+
+#define DEFINE_SAMPLE_BANK(name, medium, cachePolicy) \
+    extern u8 name##_Start[];                         \
+    extern u8 name##_Size[];
+#define DEFINE_SAMPLE_BANK_PTR(name, medium, cachePolicy) /*empty*/
+
+#include "assets/audio/samplebank_table.h"
+
+#undef DEFINE_SAMPLE_BANK
+#undef DEFINE_SAMPLE_BANK_PTR
+
+// Table header
+
+NO_REORDER AudioTableHeader sSampleBankTableHeader = {
+// The table contains the number of samplebanks, count them with the preprocessor
+#define DEFINE_SAMPLE_BANK(name, medium, cachePolicy) 1 +
+#define DEFINE_SAMPLE_BANK_PTR(name, medium, cachePolicy) 1 +
+
+#include "assets/audio/samplebank_table.h"
+
+#undef DEFINE_SAMPLE_BANK
+#undef DEFINE_SAMPLE_BANK_PTR
+    0,
```

Comment:
maybe move the `0` up? It seems closely related to the `#include "assets/audio/samplebank_table.h"`, and then we can wash our hands with `#undef` after the expression has been fully defined